jenkinsfile-runner-base-image/packager-config.yml

```
@@ -19,8 +19,10 @@ buildSettings:
         # We should check if the --waitOnExit parameter makes our fork obsolete. Will be contained after 1.0-beta-30.
         # git: https://github.com/jenkinsci/jenkinsfile-runner.git
         # branch: "1.0-beta-30"
-        git: https://github.com/SAP/stewardci-jenkinsfilerunner-image
-        branch: "jenkinsfile-runner--1.0-beta-30-steward-1"
+        git: https://github.com/romanisb/jenkinsfile-runner.git
+        branch: "pr_suppress_stdout"
```

```suggestion
        branch: "stewardci"
```

This branch includes the change with the no-build-logs option and the following required commit:
https://github.com/SAP/stewardci-jenkinsfilerunner-image/commit/121b0731c78e5624f610422bc124f613bb496961